app/js/components/breadCrumbs/breadCrumb.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Breadcrumb, BreadcrumbItem } from 'reactstrap';
+import FontAwesome from 'react-fontawesome';
+import breadCrumb from './breadCrumb.css';
+import { Link, browserHistory, withRouter } from 'react-router';
+
+class BreadCrumbComponent extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const { location, router } = this.props
+    return (
+      <div>
+        {location.pathname === '/'
+          ?
+          <Breadcrumb>
+            <BreadcrumbItem active> <a id="home" href="https://modules-refapp.openmrs.org/openmrs/index.htm"><FontAwesome className='fa fa-home' name='home' /></a>< FontAwesome name='chevron-right' className='fa fa-chevron-right' />Patient Search</BreadcrumbItem>
+          </Breadcrumb>
+          : (location.pathname.includes('/patient/') && location.pathname.includes('/visit/')) ?
+            <Breadcrumb>
+              <BreadcrumbItem id="visited" onClick={router.goBack}> <a id="home" href="https://modules-refapp.openmrs.org/openmrs/index.htm"><FontAwesome className='fa fa-home' name='home' /></a>< FontAwesome name='chevron-right' className='fa fa-chevron-right' />Patient Search</BreadcrumbItem>
```

Comment:
This block of code, as well as other lines in this file, are too long, I would recommend breaking them into smaller lines, so as to make them more readable.

Reply:
Sure. Thanks